src/meta/src/hummock/manager/mod.rs

```diff
@@ -2748,7 +2757,7 @@ async fn write_exclusive_cluster_id(
     } else {
         let cluster_id = object_store.read(&cluster_id_full_path, None).await?;
         Err(ObjectError::internal(format!(
-            "data directory is already used by another cluster with id {:?}",
+            "Data directory is already used by another cluster with id {:?}, please try again after deleting the `data_directory/cluster_id/0.data`.",
```

Comment:
Using the constants `CLUSTER_ID_DIR` and `CLUSTER_ID_NAME` in a format string can be more robust.